docs/guide/custom_policies.md

```diff
@@ -125,8 +126,7 @@ To provide support for editing custom policies in the Back Office, you need to i
 Next, register the service in DIC (Dependency Injection Container) with the `ez.limitation.formMapper` tag and set the `limitationType` attribute to the Limitation type's identifier:
```

Comment:
Not part of the changes here but, @DominikaK we kind of stopped shortening Dependency Injection Container didn't we?

Reply:
If you're referring to using the DIC acronym, then yes, we need to unify the way we refer to registering services (in most places we just say "register the service..." without spelling out DIC). To be done in some other PR throughout the whole doc.

Reply:
@andrerom, @DominikaK done.